rotkehlchen/db/dbhandler.py

```diff
@@ -3494,3 +3494,39 @@ def get_associated_locations(self) -> Set[Location]:
         if cursor.fetchone()[0] >= 1:
             locations.add(Location.BALANCER)
         return locations
+
+    def get_timed_balances(self, timestamp: Timestamp) -> List[SingleDBAssetBalance]:
+        balances_data = []
+        cursor = self.conn.cursor()
+        timed_balances_result = cursor.execute(
+            'SELECT category, time, amount, usd_value FROM timed_balances '
+            'WHERE time=?', (timestamp,),
+        )
+        for data in timed_balances_result:
+            balances_data.append(
+                SingleDBAssetBalance(
```

Comment:
Don't use SingleDBAssetBalance.

The structure to return should contain a list of balance type, asset, amount, and usd values. This one does not contain the asset.

You probably want `DBAssetBalance` but that still contains timestamp. You can probably just ignore the timestamp at export. But you should create a `serialize()` function to serialize it for CSV purposes. Don't use `process_result_list()` as that will result in the asset identifier being used which is not human readable. Instead it should be `str(asset)` for the asset, which will create a human readable represntation`.